Server/ConcordiaCurriculumManager/DTO/Dossiers/DossierDetailsWithDiscussionDTO.cs

```diff
@@ -0,0 +1,8 @@
+﻿using ConcordiaCurriculumManager.DTO.Dossiers.DossierReview;
+
+namespace ConcordiaCurriculumManager.DTO.Dossiers;
+
+public class DossierDetailsWithDiscussionDTO : DossierDetailsDTO
+{
+    public required DossierDiscussionDTO Discussion { get; set; }
```

Comment:
If we create the discussion board on initial dossier creation, then this Discussion field can just be part of the DossierDetailsDTO and this file can be removed for simplicity.